### PR TITLE
Boundary system documentation (and a minor fix to the spatial docs)

### PR DESCRIPTION
**Pull request by davidkline-ms**
Documentation for the boundary system as of RC1

### CODE REVIEW

File: Documentation/Boundary/ConfiguringBoundaryVisualization.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Configuring the Boundary Visualization
+
+The Boundary Visualization Profile provides options for configuring:
+
+- [General Settings](#general-settings)
+- [Floor Settings](#floor-settngs)
```
Comment (davidkline-ms) [inline Documentation/Boundary/ConfiguringBoundaryVisualization.md]:
```suggestion
- [Floor Settings](#floor-settings)
```

File: Documentation/Boundary/ConfiguringBoundaryVisualization.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+### Physics Layer
+
+The layer on which the boundary walls should be set. The default value is the Ignore Raycast layer.
+
+> Setting the boundary ceiling component to a physics layer other than Ignore Raycast may prevent users from interacting with objects within the scene.
+
+## See Also
+
+- [Boundary System](BoundaryGettingStarted.md)
```
Comment (davidkline-ms) [inline Documentation/Boundary/ConfiguringBoundaryVisualization.md]:
```suggestion
- [Boundary System](BoundarySystemGettingStarted.md)
```

**Comment by davidkline-ms** [comment]
addressed feedback from @wiwei and bad links detected by the ci system

File: Documentation/Boundary/BoundarySystemGettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+The Boundary system provides support for visualizing Virtual Reality boundary components in
+mixed reality applications. Boundaries define the area in which users can move around 
+while wearing a VR headset.
+
+Boundaries are an important component of a mixed reality experience to help users avoid unseen 
+obstacles while wearing a VR headset.
+
+Many Virtual Reality platforms provide an automatic display, for example a white outline 
+superimposed on the virtual world as the user or their controller nears the boundary. The
+Mixed Reality Toolkit's boundary system extends this feature to enable the diplay of an
```
Comment (thalbern) [inline Documentation/Boundary/BoundarySystemGettingStarted.md]:
```suggestion
Mixed Reality Toolkit's boundary system extends this feature to enable the display of an 
```

File: Documentation/Boundary/ConfiguringBoundaryVisualization.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+Indicates the material that should be used when creating the tracked area outline.
+
+### Physics Layer
+
+The layer on which the tracked area should be sets. The default value is the Ignore Raycast layer.
+
+## Boundary Wall Settings
+
+![Boundary Visualization Boundary Wall Settings](../../External/ReadMeImages/Boundary/BoundaryVisualizationWalllSettings.png)
```
Comment (keveleigh) [inline Documentation/Boundary/ConfiguringBoundaryVisualization.md]:
```suggestion
![Boundary Visualization Boundary Wall Settings](../../External/ReadMeImages/Boundary/BoundaryVisualizationWallSettings.png)
```

**Pull request merged**